generators/entity-server/templates/dotnetcore/src/Project.Domain/Entities/Entity.cs.ejs

```diff
@@ -88,6 +90,8 @@ namespace <%= namespace %>.Domain {
         public string UserId { get; set; } 
                 <%_ }  _%>
         public <%= otherEntityNamePascalized %> <%= relationshipFieldNamePascalized %> { get; set; }
+
+        public long? <%= relationshipFieldNamePascalized %>Id { get; set; }
```

Comment:
Is UserId still needed?

Reply:
good point, i need more test because UserId is a string

Reply:
Yes, the UserId type is string. 
The problem I see here is that line 90 defines UserId as string and line 94 also defines UserId as long.

Reply:
done